frontend/src/app/util/date-time-parser.ts

```diff
@@ -0,0 +1,17 @@
+import isSameDay from "date-fns/isSameDay";
+import format from "date-fns/format";
+import isSameMonth from "date-fns/isSameMonth";
+
+export function parseDateInReferenceToToday(date: Date): string {
```

Comment:
I'd call this `relativeDateString` or something similar. Also `parse` sounds a bit like it would do the oppostie of what it is actually doing.